src/components/FilterGroup.tsx

```diff
@@ -85,15 +95,27 @@ export function FilterGroup({
       {renderTitle()}
       <CollapsibleSection className={cssClasses.optionsContainer}>
         {searchable && <SearchInput className={cssClasses.searchInput} />}
-        {filterOptions.map(o => {
-          return (
-            <CheckboxOption
-              {...o}
-              key={o.displayName || o.value.toString()}
-              customCssClasses={cssClasses}
-            />
-          );
-        })}
+        {showAll
+          ? filterOptions.map(o => {
+            return (
+              <CheckboxOption
+                {...o}
+                key={o.displayName || o.value.toString()}
+                customCssClasses={cssClasses}
+              />
+            );
+          })
+          : filterOptions.slice(0, showMoreLimit).map(o => {
```

Comment:
since the `map` function's body is the same for both of these, can we consolidate them by moving the ternary operator within the `slice` function's params? (e.g. something like `filterOptions.slice(0, showAll ? filterOptions.length : showMoreLimit).map(...)`)

Reply:
ooooh smart ty!